home-lp/scss/includes/_header.scss

```
@@ -1,7 +1,185 @@
-.header {
-  &__title {
-    color: $title-color;
-    font-size: 36px;
-    font-weight: 500;
-  }
-}
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700;900&display=swap');
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    list-style: none;
+}
+
+body {
+}
+
+a {
+    text-decoration: none;
+    color: #fff;
+}
+
+.container {
+    margin: 0 auto;
+    max-width: 1200px;
+}
+
+.header {
+    background: #7711DD;
+    height: 882px;
+    background-image: url("../../img/bg__head-cloud.png");
+    background-position: right bottom;
+    background-repeat: no-repeat;
+
+}
+
+/*--Start Navbar Block--*/
+
+.nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 90px;
+}
+
+.nav__logo-easy {
+    color: #fff;
+    width: 52px;
+    height: 52px;
+    border-radius: 10px;
+    border: 1px solid #1C1C1D;
+    background: #1BDBC4;
+    box-shadow: 0 4px 0 0 rgb(0, 0, 0);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Permanent Marker', cursive;
+    cursor: pointer;
+}
+/*-- Bonus --*/
+.nav__logo-easy:hover{
+    color: #fff;
+    width: 55px;
+    height: 55px;
+    border-radius: 10px;
+    border: 1px solid #1C1C1D;
+    background: #1BDBC4;
+    box-shadow: 0 4px 0 0 rgb(0, 0, 0);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Permanent Marker', cursive;
+    cursor: pointer;
+    transition: all .2s;
+}

```

Краще закинути весь цей блок в середину .nav__logo-easy 
```suggestion
.nav__logo-easy {
    ..main block styles
    
    &:hover {
      ...styles for hover
    }
}

```